server/routerlicious/packages/lambdas-driver/src/kafka-service/partitionManager.ts

```diff
@@ -58,13 +58,11 @@ export class PartitionManager extends EventEmitter {
 			});
 
 			this.consumer.on("checkpoint_success", (partitionId, queuedMessage) => {
```

Comment:
> (partitionId, queuedMessage) [](http://example.com/codeflow?start=41&length=29)

Let's also log `retries` and `latency`

Reply:
Do we want to log this info on `checkpoint_error` as well?

Reply:
Thanks for including the logs for it :)